requirements/plugins.txt

```diff
@@ -2,3 +2,4 @@ torch
 tensorflow
 tensorflow_datasets
 tensorflow
+Pillow-SIMD
```

Comment:
why are you bringing this back? this fails on lots of machines

Reply:
if you really need this, you should create a `requirements/docker.txt` file

Reply:
Pillow-simd should be in plugins, it is much faster than Pillow for certain use-cases. Just like for additional functionality we have torch & tensorflow in plugins.

Also why would I create a new file docker.txt?